<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTriangle</class>
 <widget class="QDialog" name="DialogTriangle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>371</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="aspectTriangle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Внешний вид треугольника:</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>371</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="PenColor">
       <property name="text">
        <string>Цвет контура</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BrushColor">
       <property name="text">
        <string>Цвет Заливки</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="parametersTriangle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>391</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры треугольника :</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>371</width>
      <height>115</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="3">
      <widget class="QLineEdit" name="BaseLine"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="SideLabel">
       <property name="text">
        <string>Сторона:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="SideLine">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="posYLabel">
       <property name="text">
        <string>Oy:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="linePositionY"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="BaseLabel">
       <property name="text">
        <string>Основание:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="linePositionX"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="posXLabel">
       <property name="text">
        <string>Ox:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogTriangle</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogTriangle</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
